#!/usr/bin/python3
def raise_exception():
    a = "mid"
    b = 3
    c = a + b
